README.md

```diff
@@ -172,7 +168,35 @@ To install a runnable binary to `~/.cabal/bin/`:
 cabal new-install
 ```
 
-## Configuring and Running a Chainweb Node
+## Bootstrap Nodes
+
+### Testnet Nodes
+
+- us1.testnet.chainweb.com
+- us2.testnet.chainweb.com
+- eu1.testnet.chainweb.com
+- eu2.testnet.chainweb.com
+- ap1.testnet.chainweb.com
+- ap2.testnet.chainweb.com
+
+### Mainnet Nodes
+
+All bootstrap nodes are running on port 443.
+
+- us-e1.chainweb.com
+- us-e2.chainweb.com
+- us-e3.chainweb.com
+- us-w1.chainweb.com
+- us-w2.chainweb.com
+- us-w3.chainweb.com
+- jp1.chainweb.com
+- jp2.chainweb.com
+- jp3.chainweb.com
+- fr1.chainweb.com
+- fr2.chainweb.com
+- fr3.chainweb.com
+
+## Configuring, running, and monitoring the health of a Chainweb Node
```

Comment:
Would it make sense to add a subsection about using the [docker image](https://hub.docker.com/repository/docker/kadena/chainweb-node) to run a node. It's not as flexible as running the binary directly. But it's an easy way to quickly get up and running (including initializing the database).

Reply:
Its a good idea and something we should add, but we can do it a different PR

Reply:
Let's do that in a follow-up PR.  I'd like to get this in so people know about the new kadena-docs site.